python/ray/tests/test_basic.py

```diff
@@ -37,7 +37,10 @@ def test_release_resources_race(shutdown_only):
     ray.init(
         num_cpus=2,
         object_store_memory=700e6,
-        _system_config={"inline_object_status_in_refs": True})
+        _system_config={
+            "inline_object_status_in_refs": True,
+            "worker_lease_timeout_milliseconds": 1000
```

Comment:
This test seems to fail due to lease expiration without this. I don't think this should be a problem, but if it is, we should rely on microbenchmarks to catch it, not this test.